CONTRIBUTING.md

```diff
@@ -90,6 +90,28 @@ Take the steps below _before_ [running JetBrains plugin with agent](#developing-
       Note: After [#56254](https://github.com/sourcegraph/sourcegraph/issues/56254) is resolved this step is not needed
       anymore.
 
+## Updating Cody version
+
+To upgrade Cody to the newer version you need to update the `cody.commit` hash in `gradle.properties`.
+After doing that:
+
+1. Run `git diff --name-status OLD_HASH..NEW_HASH -- agent`  
```

Comment:
Can you make this instruction more precise? I tried to run it:

```
mkondratek@[pro 14] VL/D/1138  Mikołaj Kondratek scripts % git diff --name-status c8f5355a88783cce6bf1ac0088c73917e2a29d94..632887f9f0cf598579bb0b74855e6761b659d2a0 -- agent
fatal: Invalid revision range c8f5355a88783cce6bf1ac0088c73917e2a29d94..632887f9f0cf598579bb0b74855e6761b659d2a0
```

As you see, it fails 😕

Reply:
it should be ran on the Cody repo, I update the instructions.